x-pack/plugins/monitoring/public/views/base_controller.js

```diff
@@ -120,9 +120,11 @@ export class MonitoringViewBaseController {
     this.onBrush = ({ xaxis }) => {
       const { to, from } = xaxis;
       $scope.$evalAsync(() => {
```

Comment:
This and other uses of `timefilter.setTime()` shouldn't need to be wrapped in `$evalAsync` to work. Since the timefilter isn't an angular service anymore people shouldn't need angular to use it correctly. I suspect though that since some consumers are still just using getters to read the timefilter (like `kbn-chrome`) and not listening for the 'update' events, we need some other solution to make sure that timefilter updates trigger a digest cycle...